### PR TITLE
docs: changelog 5.21.3

### PR DESCRIPTION
**Pull request by MadCcc**
- 💄 Added a scroll bar to Dropdown when having many items. [#51112](https://github.com/ant-design/ant-design/pull/51112) [@Cameron-Asdf](https://github.com/Cameron-Asdf)
- 🐞 Fix Table `onChange` function receiving incorrect sorter value. [#51114](https://github.com/ant-design/ant-design/pull/51114) [@nathanlao](https://github.com/nathanlao)
- 🐞 Fix the issue about throw a warning when Splitter nested in a hidden tab panel. [#51109](https://github.com/ant-design/ant-design/pull/51109) [@kiner-tang](https://github.com/kiner-tang)
- 🐞 Fix the issue about Splitter had unexpected gaps in Flex. [#51096](https://github.com/ant-design/ant-design/pull/51096) [@kiner-tang](https://github.com/kiner-tang)
- 🐞 Add react and react-dom peerDependencies. [#51079](https://github.com/ant-design/ant-design/pull/51079) [@chentsulin](https://github.com/chentsulin)

---

- 💄 优化 Dropdown 列表较长时的滚动条样式。[#51112](https://github.com/ant-design/ant-design/pull/51112) [@Cameron-Asdf](https://github.com/Cameron-Asdf)
- 🐞 修复 Table 组件在切换页面时 `onChange` 函数接收到错误的 sorter 值的问题。[#51114](https://github.com/ant-design/ant-design/pull/51114) [@nathanlao](https://github.com/nathanlao)
- 🐞 修复 Splitter 嵌套在一个隐藏的 Tabs 面板中时抛出警告的问题。[#51109](https://github.com/ant-design/ant-design/pull/51109) [@kiner-tang](https://github.com/kiner-tang)
- 🐞 修复 Splitter 组件在 Flex 组件下时出现异常间距的问题。[#51096](https://github.com/ant-design/ant-design/pull/51096) [@kiner-tang](https://github.com/kiner-tang)
- 🐞 将 react 和 react-dom 添加进 peerDependencies。[#51079](https://github.com/ant-design/ant-design/pull/51079) [@chentsulin](https://github.com/chentsulin)

[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [ ] 🐞 Bug fix
- [ ] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [ ] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 🔗 Related Issues

> - Describe the source of related requirements, such as links to relevant issue discussions.
> - For example: close #xxxx, fix #xxxx

### 💡 Background and Solution

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) like a cat tracks a laser pointer.
> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### CODE REVIEW

**Review by li-jia-nan — DISMISSED** [review]

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -16,6 +16,16 @@ tag: vVERSION
...
 
+## 5.21.3
+
+`2024-10-08`
+
+- 💄 Added a scroll bar to Dropdown when having many items. [#51112](https://github.com/ant-design/ant-design/pull/51112) [@Cameron-Asdf](https://github.com/Cameron-Asdf)
+- 🐞 Fix Table `onChange` function receiving incorrect sorter value. [#51114](https://github.com/ant-design/ant-design/pull/51114) [@nathanlao](https://github.com/nathanlao)
+- 🐞 Fix the issue about throw a warning when Splitter nested in a hidden tab panel. [#51109](https://github.com/ant-design/ant-design/pull/51109) [@kiner-tang](https://github.com/kiner-tang)
+- 🐞 Fix the issue about Splitter had unexpected gaps in Flex. [#51096](https://github.com/ant-design/ant-design/pull/51096) [@kiner-tang](https://github.com/kiner-tang)
+- 🐞 MISC: Add react and react-dom peerDependencies. [#51079](https://github.com/ant-design/ant-design/pull/51079) [@chentsulin](https://github.com/chentsulin)
```
Comment (PeachScript) [inline CHANGELOG.en-US.md:27]: 本质是修复，`add` 感觉有点怪，`rollback` or `restore`？

以及翻了下以前的 changelog，包名好像会用 <code>\`react\`</code> 包起来

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -16,6 +16,16 @@ tag: vVERSION
 
 ---
 
+## 5.21.3
+
+`2024-10-08`
```
Comment (li-jia-nan) [inline CHANGELOG.en-US.md:21]:
```suggestion
`2024-10-09`
```

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,16 @@ tag: vVERSION
 
 ---
 
+## 5.21.3
+
+`2024-10-08`
```
Comment (li-jia-nan) [inline CHANGELOG.zh-CN.md:20]:
```suggestion
`2024-10-09`
```

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -15,6 +15,16 @@ tag: vVERSION
...
 
+## 5.21.3
+
+`2024-10-09`
+
+- 💄 优化 Dropdown 列表较长时的滚动条样式。[#51112](https://github.com/ant-design/ant-design/pull/51112) [@Cameron-Asdf](https://github.com/Cameron-Asdf)
+- 🐞 修复 Table 组件在切换页面时 `onChange` 函数接收到错误的 sorter 值的问题。[#51114](https://github.com/ant-design/ant-design/pull/51114) [@nathanlao](https://github.com/nathanlao)
+- 🐞 修复 Splitter 嵌套在一个隐藏的 Tabs 面板中时抛出警告的问题。[#51109](https://github.com/ant-design/ant-design/pull/51109) [@kiner-tang](https://github.com/kiner-tang)
+- 🐞 修复 Splitter 组件在 Flex 组件下时出现异常间距的问题。[#51096](https://github.com/ant-design/ant-design/pull/51096) [@kiner-tang](https://github.com/kiner-tang)
+- 🐞 杂项：将 react 和 react-dom 添加进 peerDependencies。[#51079](https://github.com/ant-design/ant-design/pull/51079) [@chentsulin](https://github.com/chentsulin)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:26]: master 还有几个新改动。

**Review by kiner-tang — DISMISSED** [review]

File: CHANGELOG.zh-CN.md (hunk 4 above)
Comment (afc163) [reply CHANGELOG.zh-CN.md:26]: https://github.com/ant-design/ant-design/pull/51161

Comment (afc163) [reply CHANGELOG.zh-CN.md:26]: https://github.com/ant-design/ant-design/pull/51150 这里还有两个。

**Review by li-jia-nan — DISMISSED** [review]

**Review by zombieJ — DISMISSED** [review]

File: CHANGELOG.zh-CN.md [hunk 5]
```diff
@@ -15,6 +15,18 @@ tag: vVERSION
...
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:26]: 英文已经改成 restore 了，中文也改一下吧。

**Review by li-jia-nan — DISMISSED** [review]

File: CHANGELOG.zh-CN.md [hunk 6]
```diff
@@ -15,6 +15,22 @@ tag: vVERSION
 
 ---
 
+## 5.21.3
+
+`2024-10-09`
+
+- 💄 优化 Dropdown 列表较长时的滚动条样式。[#51112](https://github.com/ant-design/ant-design/pull/51112) [@Cameron-Asdf](https://github.com/Cameron-Asdf)
+- Slider [#51150](https://github.com/ant-design/ant-design/pull/51150) [@yoyo837](https://github.com/yoyo837)
+  - 🐞 修复 Slider 不支持 id 属性的问题。
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:24]:
```suggestion
  - 🐞 修复 Slider 不支持 `id` 属性的问题。
```

File: CHANGELOG.zh-CN.md [hunk 7]
```diff
@@ -15,6 +15,22 @@ tag: vVERSION
...
+  - 🐞 修复 Slider 导致 extractStyle 时抛出 useLayoutEffect does nothing on the server 警告信息的问题。
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:25]:
```suggestion
  - 🐞 修复 Slider 导致 `extractStyle` 时抛出 `useLayoutEffect does nothing on the server` 警告信息的问题。
```

File: CHANGELOG.zh-CN.md [hunk 8]
```diff
@@ -15,6 +15,22 @@ tag: vVERSION
...
+- 🐞 修复 ColorPicker 渐变色时，部分节点颜色拖拽会被强制重置为第一个节点颜色的问题。[#51161](https://github.com/ant-design/ant-design/pull/51161) [@zombieJ](https://github.com/zombieJ)
+- 🐞 修复 Table 组件在切换页面时 `onChange` 函数接收到错误的 sorter 值的问题。[#51114](https://github.com/ant-design/ant-design/pull/51114) [@nathanlao](https://github.com/nathanlao)
+- 🐞 修复 Splitter 嵌套在一个隐藏的 Tabs 面板中时抛出警告的问题。[#51109](https://github.com/ant-design/ant-design/pull/51109) [@kiner-tang](https://github.com/kiner-tang)
+- 🐞 修复 Splitter 组件在 Flex 组件下时出现异常间距的问题。[#51096](https://github.com/ant-design/ant-design/pull/51096) [@kiner-tang](https://github.com/kiner-tang)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:29]: Splitter 合并一下？

File: CHANGELOG.en-US.md [hunk 9]
```diff
@@ -16,6 +16,22 @@ tag: vVERSION
...
 
+## 5.21.3
+
+`2024-10-09`
+
+- 💄 Added a scroll bar to Dropdown when having many items. [#51112](https://github.com/ant-design/ant-design/pull/51112) [@Cameron-Asdf](https://github.com/Cameron-Asdf)
+- Slider [#51150](https://github.com/ant-design/ant-design/pull/51150) [@yoyo837](https://github.com/yoyo837)
+  - 🐞 Fix Slider issue where the `id` prop is not supported.
+  - 🐞 Fix Slider to address the issue causing `useLayoutEffect` does nothing on the server warning when `extractStyle` is invoked.
+- 🐞 Fix ColorPicker with gradient mode, sometime handle color will be force sync back to first handle color issue. [#51161](https://github.com/ant-design/ant-design/pull/51161) [@zombieJ](https://github.com/zombieJ)
```
Comment (afc163) [inline CHANGELOG.en-US.md:27]:
```suggestion
- 🐞 Fix ColorPicker with gradient mode, sometimes handle color will be force sync back to first handle color issue. [#51161](https://github.com/ant-design/ant-design/pull/51161) [@zombieJ](https://github.com/zombieJ)
```

File: CHANGELOG.en-US.md [hunk 10]
```diff
@@ -16,6 +16,22 @@ tag: vVERSION
...
 ---
 
+## 5.21.3
+
+`2024-10-09`
+
+- 💄 Added a scroll bar to Dropdown when having many items. [#51112](https://github.com/ant-design/ant-design/pull/51112) [@Cameron-Asdf](https://github.com/Cameron-Asdf)
+- Slider [#51150](https://github.com/ant-design/ant-design/pull/51150) [@yoyo837](https://github.com/yoyo837)
+  - 🐞 Fix Slider issue where the `id` prop is not supported.
+  - 🐞 Fix Slider to address the issue causing `useLayoutEffect` does nothing on the server warning when `extractStyle` is invoked.
```
Comment (afc163) [inline CHANGELOG.en-US.md:26]:
```suggestion
  - 🐞 Fix Slider to address the issue causing `useLayoutEffect does nothing on the server` warning when `extractStyle` is invoked.
```

**Review by afc163 — APPROVED** [review]

**Review by zombieJ — APPROVED** [review]

**Pull request merged**